pkg/asset/manifests/utils.go

```diff
@@ -34,7 +34,7 @@ func configMap(namespace, name string, data genericData) *configurationObject {
 }
 
 func getAPIServerURL(ic *types.InstallConfig) string {
-	return fmt.Sprintf("https://api.%s:6443", ic.ClusterDomain())
+	return fmt.Sprintf("https://api.%s:443", ic.ClusterDomain())
```

Comment:
ditto? default the port if it is 443? `https://api.%s`

Reply:
If you don't put the 443 some deep golang library gets mad that it doesn't know what port to connect to. Easier to just leave it than figure our why.